src/Sylius/Component/Core/Model/ProductVariant.php

```diff
@@ -78,7 +78,7 @@ public function __construct()
 
     public function __toString(): string
     {
```

Comment:
What is the point to use magic methods instead of standard method with configured logic?